src/polyhedron.jl

```diff
@@ -0,0 +1,189 @@
+export ConvexHullLib
+
+type ConvexHullLib <: PolyhedraLibrary
+    precision::Symbol
+
+    function ConvexHullLib(precision::Symbol=:float)
+        if !(precision in [:float, :exact])
+            error("Invalid precision, it should be :float or :exact")
+        end
+        new(precision)
+    end
+end
+
+type ConvexHullPolyhedron{N, T} <: Polyhedron{N, T}
+    ine::Nullable{HRepresentation{N, T}}
+    inel::Nullable{LiftedHRepresentation{N, T}}
+    ext::Nullable{VRepresentation{N, T}}
+    extl::Nullable{LiftedVRepresentation{N, T}}
+    hlinearitydetected::Bool
+    vlinearitydetected::Bool
+    noredundantinequality::Bool
+    noredundantgenerator::Bool
+
+    function ConvexHullPolyhedron(ine::HRepresentation{N, T}, ext::VRepresentation{N, T}, hld::Bool, vld::Bool, nri::Bool, nrg::Bool)
+        new(ine, nothing, ext, nothing, hld, vld, nri, nrg)
+    end
+    function ConvexHullPolyhedron(ine::HRepresentation{N, T})
+        new(ine, nothing, nothing, nothing, false, false, false, false)
```

Comment:
I guess ``nothing`` gets converted to ``Nullable{VRepresentation{N,T}}()``? Might be better to make this explicit.